kafka/consumer_test.go

```diff
@@ -907,8 +907,6 @@ func assertNotNilOptions(t testing.TB, cfg *ConsumerConfig) {
 	cfg.Processor = nil
 	assert.NotNil(t, cfg.Logger)
 	cfg.Logger = nil
-	assert.NotNil(t, cfg.TopicAttributeFunc)
-	cfg.TopicAttributeFunc = nil
```

Comment:
Since we wrap this function with multiple attributes, this is no longer valid, so I removed the check